libraries/AP_GPS/AP_GPS.h

```diff
@@ -135,13 +135,17 @@ class AP_GPS
         uint8_t num_sats;                   ///< Number of visible satellites
         Vector3f velocity;                  ///< 3D velocity in m/s, in NED format
         float speed_accuracy;               ///< 3D velocity accuracy estimate in m/s
-        float horizontal_accuracy;          ///< horizontal accuracy estimate in m
-        float vertical_accuracy;            ///< vertical accuracy estimate in m
+        float horizontal_accuracy;          ///< horizontal accuracy estimate in m RMS
```

Comment:
I'm not sure this is actually true for all the instances. U-blox doesn't note it formally either way in their protocol data sheet (that I can find), but they cite the accuracies as CEP when stating the performance of the unit, which I suspect means they might stick with that reporting for real time reasons as well since it would correspond with the data sheet better.